oaknational/Oak-Web-Application
# Feat/pupil 901/quiz result match

## Description

Music year: 1955

- List of changes

### Job Story

**When** a question has type “match”,
**I want to a component** OakQuizMatchResult
**So that the results of the question can be displayed**.

## Issue(s)

Fixes #

## How to test

1. Go to [https://deploy-preview-2662--oak-web-application.netlify.thenational.academy](https://deploy-preview-2662--owa-storybook.netlify.thenational.academy/?path=/story/components-pupilcomponents-quizresultmatch--correct-match)
2. Click on correct and incorrect answers
3. You should see correct style and copy

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
